coalib/bears/requirements/PackageRequirement.py

```diff
@@ -25,6 +34,12 @@ def check(self):
         """
         Check if the requirement is satisfied.
 
+        >>> PR = PackageRequirement('pip', 'coala_decorators', '0.2.1')
+        >>> PR.check()
```

Comment:
if it fits into a single line, you could inline:
`PackageRequirement(...).check()`